### DISCUSSION TITLE
Data export functionality with search and filter

### DISCUSSION THREAD

**Discussion by iotron** [orchidsoftware/platform#1854] [Ideas]
A way to persist the data, after filter and search, so that it can be exported from a command bar button with a simple CSV export functionality.
https://github.com/orchidsoftware/platform/blob/902925c644bbf1516a815292cad40805e249316c/src/Screen/Screen.php#L62

**Comment by tabuna** [comment]
On transition, this value will still be cleared. Since this is a new request. Use something like:

```php
Button::make('Export')->parameters(
    request()->all()
);
```
